packages/nouns-webapp/src/utils/moderation/containsBlockedText.ts

```diff
@@ -0,0 +1,28 @@
+import moderationRegexes from './moderationRegexes.json';
+
+/**
+ * Check if text matches a blocked phrase in a given langugae
+ *
+ * @param text
+ * @param language
+ * @returns boolean true iif text matches a blocked word of phrase
+ */
+export const containsBlockedText = (text: string, language: string) => {
```

Comment:
Thoughts on being slightly more proactive and using https://github.com/web-mech/badwords as a base with some custom words included. The fallback of showing an address makes accidental detection very low-stakes.